operating_unit/view/res_users_view.xml

```diff
@@ -8,7 +8,7 @@
             <xpath expr="//page[@name='access_rights']/group[1]" position="after">
                 <group
                     string="Operating Units"
-                    groups="operating_unit.group_multi_operating_unit"
+                    groups="operating_unit.group_user_operating_unit"
```

Comment:
Maybe it should be the manager the one that can change OUs in the user, don't you think?